docs/consuming-prebuilt-podvm-images.md

```diff
@@ -0,0 +1,14 @@
+# Consuming pre-built podvm images
+
+The project has automation in place to build container images that contain the qcow2 file for the various Linux distributions and cloud providers supported. Those images are re-built after each pull request are merged into this repository.
+
+You can find the images available at https://quay.io/organization/confidential-containers with the *podvm-[CLOUD_PROVIDER]-[DISTRO]* name pattern. For example, https://quay.io/repository/confidential-containers/podvm-libvirt-ubuntu hosts the Ubuntu images for Libvirt.
+
+The easiest way to extract the qcow2 file from the podvm container image is using the [`extract-qcow2.sh`](../podvm/hack/extract-qcow2.sh) script. For example, to extract the file from the *podvm-libvirt-ubuntu* image:
+
+```
+$ cd podvm
+$ ./hack/extract-qcow2.sh quay.io/confidential-containers/podvm-libvirt-ubuntu
+```
+
+If you need to customize the image, for example, to add a [registries authentication configuration](registries-authentication.md) then it can be done with the [libguestfs](https://www.libguestfs.org) tools. Alternatively, you can find instructions on [podvm/README.md](../podvm/README.md) on how to build the image from sources.
```

Comment:
can you use [statically embedded registries authentication configuration](registries-authentication.md#statically-embed-authentication-file-in-podvm-image) instead? as it's not needed if auth file passed at runtime
raw:
`[statically embedded registries authentication configuration](registries-authentication.md#statically-embed-authentication-file-in-podvm-image)`

Reply:
I am still a little confused here but I gave it a try and re-wrote that section. Let me know if it looks good (or not).